### DISCUSSION TITLE
New User & Need Direction

### DISCUSSION THREAD

**Discussion by 05201968** [community/community#34130] [Other Feature Feedback, Questions, & Ideas]
Hi! I'm very new to GitHub and it feels like a different world. I'm here because I'm interested in the capabilities of all that this offers. I'm particularly interested in keeping myself and others safe and protecting intellectual property. I have a long way to go to understand, learn and actually  provide help to others that have had the same privacy invasion that I have. Hopefully , I can made some good out of the bad if focus and listen to others. So, I'm looking for guidance please. I am open to conversation and direction. Ty

**Comment by DylanDevelops** [comment]
Hopefully, this can help,

 If you are looking to protect your projects from being resold without giving you credit, licenses will be your friend. When creating a repository you can add a license. Research different types of licenses and see which ones fit your project's needs the best. 

For making sure you are safe, be careful with what is uploaded to GitHub when doing pushes, especially when pushing to a public repository. You do not want to store sensitive information inside of source code that is being uploaded to a public repository as that code can be seen by anyone on the internet.

If you have any questions let me know and have fun with GitHub! There is a lot you can do with the program and it will make your life 10x easier!

-Dylan
